lib/Command/ExportUser.php

```diff
@@ -42,14 +43,16 @@ protected function configure() {
 		$this->setName('instance:export:user')
 			->setDescription('Exports a single user')
 			->addArgument('userId', InputArgument::REQUIRED, 'User to export')
-			->addArgument('exportDirectory', InputArgument::REQUIRED, 'Path to the directory to export data to');
+			->addArgument('exportDirectory', InputArgument::REQUIRED, 'Path to the directory to export data to')
+			->addOption("no-files", "m", InputOption::VALUE_NONE, 'Skip exporting file');
 	}
```

Comment:
"no-files", "n"?

Reply:
The thinking was "no-file" -m -> metadata... but yeah maybe too far fetched...

Reply:
@micbar "n" is already reserved by symfony for --no-interaction

Reply:
ah, ok